api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -93,6 +93,10 @@
   private static final String SAM_STATUS_NAME = "Sam";
   private static final String RAWLS_STATUS_NAME = "Rawls";
   private static final String GOOGLE_BUCKETS_STATUS_NAME = "GoogleBuckets";
+  // The defaul location for AoU buckets. Setting this location when cloning workspaces to avoid
```

Comment:
please clean up the grammar here.

Also, the URL doesn't resolve for me.  Is that an internal Google link?